test/integration/results.html.tmpl

```diff
@@ -1,76 +1,73 @@
-    if (el.tagName === 'IMG' && el.dataset.altSrc) {
-      var tmp = el.src;
-      el.src = el.dataset.altSrc;
-      el.dataset.altSrc = tmp;
-    }
-  }
-</script>
-<% if (shuffle) { %>
-<div>Shuffle seed: <%- seed %></div>
-<% } %>
 <% if (unsuccessful.length) { %>
-<div>
-  <strong style="color: darkred;">Failed Tests:</strong>
-  <% for (const r of unsuccessful) { %><%- r.id %> <% } %>
-</div>
+<h1 style="color: red;">
+    <%- unsuccessful.length %> tests failed.
 <% } else { %>
-<div><strong style="color: darkgreen;">All tests passed!</div>
+<h1 style="color: green;">
+    All tests passed!
 <% } %>
-<% if (tests.length) { %>
-<button id='toggle-sequence'>Toggle showing test sequence</button>
-<div id='test-sequence' style='display:none'>
-    <strong style="color: black;">Test sequence:</strong>
-    <% for (const s of tests) { %><%- s.id %> <% } %>
+    <button id='toggle-sequence'>Toggle test sequence</button>
+    <button id='toggle-passed'>Toggle passed tests</button>
+    <button id='toggle-ignored'>Toggle ignored tests</button>
+</h1>
+
+<p class="stats"><%- Object.keys(stats).map(status => stats[status] + ' ' + status).join(', ') + '.' %></p>
+
+<div id='test-sequence' class='hide'>
+    <% if (unsuccessful.length) { %>
+    <p><strong>Failed tests:</strong>
+    <% for (const r of unsuccessful) { %><%- r.id %> <% } %></p>
+    <% } %>
+
+    <p><strong>Test sequence:</strong>
+    <% for (const s of tests) { %><%- s.id %> <% } %></p>
+
+    <% if (shuffle) { %><p><strong>Shuffle seed</strong>: <%- seed %></p><% } %>
 </div>
-<% } %>
-<button id='toggle-passed'>Toggle showing passed tests</button>
-<button id='toggle-ignored'>Toggle showing ignored tests</button>
+
 <script>
-const passedButton = document.getElementById('toggle-passed');
-passedButton.addEventListener('click', function (e) {
-  const passed = document.querySelectorAll('tr.passed');
-  for (const row of passed) {
-    row.classList.toggle('hide');
-  }
-});
+document.addEventListener('mouseover', handleHover);
+document.addEventListener('mouseout', handleHover);
 
-const ignoredButton = document.getElementById('toggle-ignored');
-ignoredButton.addEventListener('click', function (e) {
-  const ignored = document.querySelectorAll('tr.ignored');
-  for (const row of ignored) {
-    row.classList.toggle('hide');
-  }
-});
+function handleHover(e) {
+    var el = e.target;
+    if (el.tagName === 'IMG' && el.dataset.altSrc) {
+        var tmp = el.src;
+        el.src = el.dataset.altSrc;
+        el.dataset.altSrc = tmp;
+    }
+}
 
-const sequenceButton = document.getElementById('toggle-sequence');
-sequenceButton.addEventListener('click', function (e) {
-    const sequence = document.getElementById('test-sequence');
-    sequence.style.display = sequence.style.display == "none" ? "block" : "none";
+document.getElementById('toggle-passed').addEventListener('click', function (e) {
+    for (const row of document.querySelectorAll('.test.passed')) {
```

Comment:
Should this also hide `ignored passed` tests?

Reply:
@asheemmamoowala these test will have two classes — `ignored` and `passed`, so the selector above will include them.